docs/elements.md

```diff
@@ -1193,46 +1193,24 @@ def generate(data):
+| `suffix`             | string  | (empty) | Any suffix to append to the output in `text` mode.                                                                                                                                                                                                                          |
+| `indent`             | integer | 1       | Specifies the amount of indentation added for each nesting level when printing nested objects.                                                                                                                                                                              |
+| `depth`              | integer | -       | The number of nesting levels which may be printed; if the data structure being printed is too deep, the next contained level is replaced by ... By default, there is no constraint on the depth of the objects being formatted.                                             |
+| `width`              | integer | 80      | Specifies the desired maximum number of characters per line in the output. If a structure cannot be formatted within the width constraint, a best effort will be made.                                                                                                      |
+| `compact`            | boolean | false   | Impacts the way that long sequences (lists, tuples, sets, etc.) are formatted. If compact is false then each item of a sequence will be formatted on a separate line. If compact is true, as many items as will fit within the width will be formatted on each output line. |
+| `underscore-numbers` | boolean | false   | If true, integers will be formatted with the `_` character for a thousands separator, otherwise underscores are not displayed.                                                                                                                                              |
+| `sort-dicts`         | boolean | true    | If true, dictionaries will be formatted with their keys sorted, otherwise they will display in insertion order.                                                                                                                                                             |
+| `no-highlight`       | boolean | false   | Disable syntax highlighting.                                                                                                                                                                                                                                                |
 
 #### Details
 
-As of right now, the element supports displaying either Pandas DataFrames as an HTML table or Python objects via `repr()`. When setting a parameter to a DataFrame, use PrairieLearn's built in `pl.to_json()`.
```

Comment:
Realizing that this isn't quite accurate, but I think that for all cases this was being used for, the `__repr__` was equivalent to the printed output. Will double check this though.

Reply:
Is there a way to reference the `question.md` file from here?

Reply:
Something like:

```md
See [Question Writing](question.md#section-name-here)
```

Many examples of cross-referencing documentation pages can be found in the [FAQ Source](https://raw.githubusercontent.com/PrairieLearn/PrairieLearn/master/docs/faq.md) (the [FAQ page](https://prairielearn.readthedocs.io/en/latest/faq/) should show how the link appears.)

Reply:
Added this reference!